addons/medical_treatment/stringtable.xml

```diff
@@ -3699,6 +3702,12 @@
             <Chinese>%1 沒有反應</Chinese>
             <Turkish>%1 tepki vermiyor</Turkish>
         </Key>
+        <Key ID="STR_ACE_Medical_Treatment_Check_Response_CardiacArrest">
+            <English>%1 is not responsive, taking shallow gasps and convulsing</English>
```

Comment:
Gasping is fairly common in early cardiac arrest, not sure if convulsing is. I am basing this on civilian first aid training (that I luckily have not had to use yet), not military experience.